src/__screenshot_tests__/poster-card-screenshot-test.tsx

```diff
@@ -0,0 +1,101 @@
+import {openStoryPage, screen, setRootFontSize} from '../test-utils';
```

Comment:
Needless to say, I've taken the idea from other similar components. The really only new one I've added is commented below